apps/fluent-tester/src/FluentTester/TestComponents/ButtonExperimental/ButtonVariantTestSection.tsx

```diff
@@ -1,20 +1,58 @@
 import { Button, CompoundButton } from '@fluentui-react-native/experimental-button';
 import * as React from 'react';
-import { View } from 'react-native';
+import { View, Platform, processColor } from 'react-native';
 import { commonTestStyles, stackStyle } from '../Common/styles';
 
 export const ButtonVariantTest: React.FunctionComponent<{}> = () => {
-  return (
-    <View style={[stackStyle, commonTestStyles.view]}>
-      <Button content="Default" style={commonTestStyles.vmargin} />
-      <Button primary content="Primary" style={commonTestStyles.vmargin} />
-      <Button ghost content="Ghost" style={commonTestStyles.vmargin} />
-      <Button fluid content="Fluid" style={commonTestStyles.vmargin} />
-      <Button primary fluid content="Fluid Primary" style={commonTestStyles.vmargin} />
-      <Button ghost fluid content="Fluid Ghost" style={commonTestStyles.vmargin} />
-      {/* <CompoundButton content="Default" secondaryContent="Compound" style={commonTestStyles.vmargin} />
-      <CompoundButton primary content="Primary" secondaryContent="Compound" style={commonTestStyles.vmargin} /> */}
-      <CompoundButton ghost content="Ghost" secondaryContent="Compound" style={commonTestStyles.vmargin} />
-    </View>
-  );
+  if (Platform.OS == 'macos') {
+    const CustomizedNativeButton = Button.customize({
+      contentTintColor: processColor('white'),
+      restBackgroundColor: processColor('orange'),
```

Comment:
I feel these names are very specific to the macOS implementation and should not be in the React Native Button. I feel there should be a more general name (one that probably comes from design like `accentColor`) that maps to `contentTintColor` internally when the JS component passes props to the native module.